### PR TITLE
Singular active and blooming periods

### PR DESCRIPTION
**Pull request by floogulinc**

### CODE REVIEW

**Comment by floogulinc** [comment]
The regex I used for the find and replace if anyone is interested:
```regex
\[[\n\r\s]+"--\d\d-\d\d/--\d\d-\d\d"[\n\r\s]+\]
```

**Review by helloworld12321 — APPROVED** [review]
Looks good—thanks, Paul! :)

File: bees.json [hunk 1]
```diff
@@ -178,10 +172,7 @@
           "image_link": "https://www.inaturalist.org/photos/4965870"
         }
       ],
-      "active_period": [
-        "--04-01/--05-31",
-        "--09-01/--09-30"
-      ],
+      "active_period": "--04-01/--06-31",
```
Comment (kklamberty) [inline bees.json:175]: This should be changed. 6-31 is not a date.

Comment (floogulinc) [reply bees.json:175]:
```suggestion
      "active_period": "--04-01/--06-30",
```

Comment (floogulinc) [reply bees.json:175]: Fixed in https://github.com/mn-pollinators/assets/commit/cd27ab7631c413ccb96fc99384314febf4b1680f

**Pull request merged**